src/__tests__/fireEvent.test.js

```diff
@@ -57,7 +57,9 @@ describe('fireEvent', () => {
   });
 
   test('should throw an Error when event handler was not found', () => {
-    const { getByTestId } = render(<WithoutEventComponent />);
+    const { getByTestId } = render(
+      <WithoutEventComponent onPress={() => 'this is not passed to children'} />
```

Comment:
Based on https://github.com/callstack/react-native-testing-library/issues/28 I've added this instead of the `should not bubble event to root element` test that started to fail